### PR TITLE
Bug 1146825 - Make gaia-grid use its container size

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fe745c6f6b702712faf6eca303200c80d195bcba

File: apps/search/style/search.css [hunk 1]
```diff
@@ -222,6 +222,7 @@ section.apps {
   /* Hide the whitespace between inline elements */
   font-size: 0;
   height: 10rem;
+  width: 32rem;
```
Comment (KevinGrandon) [inline apps/search/style/search.css]: I think we probably want 100% here instead if possible?

Comment (daleharvey) [inline apps/search/style/search.css]: Nope, the point was in landscape mode we want to restrict the width to the same as portrait mode so the icons are displayed at the same size

Ultimitely I think we want to have it 100% and horizontal scroll to the results, but I think thats a reasonably large patch so this one restricts us to 4 results that are displayed the same horizontal or vertical

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5dba029108a2b06addc20315b8d08df1453efb69

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8476832cfe264a622ea77a6c47c63d49b05dd4f7

**Pull request closed without merging**